client/src/amplify-ui/cli-feature-flag/feature-flags.json

```diff
@@ -42,6 +42,126 @@
             "defaultExistingProject": false
           }
         ]
+      },
+      "useExperimentalPipelinedTransformer": {
+        "description": "Use pipeline feature in new Transformer.",
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "4.32.4",
+        "deprecationDate": "May 1st 2021",
+        "values": [
+          {
+            "value": "true",
+            "description": "Allows to use pipeline feature in transfomer.",
+            "defaultNewProject": true,
+            "defaultExistingProject": false
+          },
+          {
+            "value": "false",
+            "description": "previous graphql transformer.",
+            "defaultNewProject": false,
+            "defaultExistingProject": true
+          }
+        ]
+      },
+      "enableIterativeGsiUpdates": {
+        "description": "Enable updating more than 1 GSIs in update stage .",
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "4.38.0",
+        "deprecationDate": "May 1st 2021",
+        "values": [
+          {
+            "value": "true",
+            "description": "Projects can mutate more than 1 GSIs in update flow.",
+            "defaultNewProject": true,
+            "defaultExistingProject": false
+          },
+          {
+            "value": "false",
+            "description": "Throws Error when mutating more than 1 GSIs in update flow. .",
+            "defaultNewProject": false,
+            "defaultExistingProject": true
+          }
+        ]
+      }
+    }
+  },
+  "frontend-ios": {
+    "description": "Feature Flag related to Frontend Ios",
+    "features": {
+      "enableXcodeIntegration": {
+        "description": "Enable xCode integration .",
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "4.40.0",
+        "deprecationDate": "May 1st 2021",
+        "values": [
+          {
+            "value": "true",
+            "description": "Enables Xcode integration.",
```

Comment:
All newly generated DataStore/API models are auto-added to the Xcode project.

Reply:
done!